OutOfSchool/OutOfSchool.DataAccess/Models/Configurations/TagConfiguration.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace OutOfSchool.Services.Models.Configurations;
+public class TagConfiguration: IEntityTypeConfiguration<Tag>
+{
+    public void Configure(EntityTypeBuilder<Tag> builder)
+    {
+        builder.HasKey(x => x.Id);
+
```

Comment:
I just wonder - why there are no max length limits for string fields?

Reply:
Tag is never initiated by the user, therefore there is no need for such constraints there as tags are only hardcoded apparently.